collections/item.go

```diff
@@ -6,11 +6,15 @@ import (
 	"fmt"
 
 	"cosmossdk.io/collections/codec"
+	"cosmossdk.io/server/v2/stf"
```

Comment:
This seems problematic, collections should not depend on server v2 and STF.

Reply:
Ty, Wrapped in a `container.Service`. Now collections is not depended on `stf`.